docs/docs/themes/using-a-gatsby-theme.md

```diff
@@ -2,71 +2,55 @@
-gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
+```js
+npm install --save gatsby-theme-blog
 ```
 
-## What does a theme starter do?
+## Theme Options
 
-The starter for the official Gatsby blog theme does the following:
+Depending on the theme, there may be theme options that can be configured via `gatsby-config.js`.
 
-### 1. The starter installs the theme and configures it
+For example, `gatsby-theme-blog` can take in 4 potential options: `basePath`, `contentPath`, `assetPath`, and `mdx`.
 
-When you use a starter that's built with a theme, you will often see that you're initially presented with a lighter weight `gatsby-config.js`. Themes start doing their magic when installed via the `plugins` array:
-
-```javascript:title=gatsby-config.js
+```js
+// gatsby-config.js
 module.exports = {
   plugins: [
     {
-      resolve: "gatsby-theme-blog",
-      options: {},
+      resolve: `gatsby-theme-blog`,
+      options: {
+        /*
+        - basePath defaults to `/`
+        - contentPath defaults to `content/posts`
+        - assetPath defaults to `content/assets`
+        - mdx defaults to `true`
+        */
+        basePath: `/blog`,
+        contentPath: `content/blogPosts`,
+        assetPath: `content/blogAssets`,
+        mdx: false,
+      },
     },
   ],
-  // Customize your site metadata:
-  siteMetadata: {
-    title: "My Blog Title",
-    author: "My Name",
-    description: "My site description...",
-    siteUrl: "https://www.gatsbyjs.org/",
-    social: [
-      {
-        name: "twitter",
-        url: "https://twitter.com/gatsbyjs",
-      },
-      {
-        name: "github",
-        url: "https://github.com/gatsbyjs",
-      },
-    ],
-  },
 }
 ```
 
-### 2. The starter scaffolds out example blog posts.
-
-```md:title=/content/posts/hello-world.mdx
----
-title: Hello, world!
-path: /hello-world
----
-
-I'm an example post!
-```
+To further customize a theme, see [Gatsby theme shadowing](/docs/themes/shadowing/).
 
-Once you've made some edits, run `gatsby develop` to start a development server and view your changes in a browser.
+## Published Themes
 
-## Updating a theme
+Public Gatsby themes are published on npm for anyone to use. You can also publish private themes for use by your organization. Examples of private npm package hosting include [npm registry](https://docs.npmjs.com/about-private-packages) and [GitHub Package Registry](https://help.github.com/en/github/managing-packages-with-github-package-registry/about-github-package-registry).
 
-To pull in the latest updates of your theme you can update the `gatsby-theme-blog` version in your site's `package.json`.
+## Using Yarn Workspaces
 
-You can do this by running the install of the theme package again: `npm install --save gatsby-theme-blog`.
+If you would like to work with unpublished themes, consider [setting up Yarn Workspaces for theme development](/blog/2019-05-22-setting-up-yarn-workspaces-for-theme-development/#reach-skip-nav).
```

Comment:
```suggestion
If you would like to work with unpublished themes, consider [setting up Yarn Workspaces for theme development](/blog/2019-05-22-setting-up-yarn-workspaces-for-theme-development/) and [using Yarn](/docs/gatsby-cli/#how-to-change-your-default-package-manager-for-your-next-project) instead of npm.
```